unmock/core/options.py

```diff
@@ -23,20 +23,8 @@ def __init__(self, save: Union[bool, List[str]] = False, unmock_host: str = UNMO
                  ignore=None, signature: Optional[str] = None, token: Optional[str] = None,
                  whitelist: Optional[List[str]] = None):
         if logger is None:
-            # TODO - move the logging definition elsewhere? Console output by default?
-            logger = logging.getLogger("reporter")
-            logger.setLevel(logging.INFO)
-            # For now, make sure we only have one stream logger...
-            has_stream_handler = False
-            for handler in logger.handlers:
-                if isinstance(handler, logging.StreamHandler):
-                    has_stream_handler = True
-                    break
-            if not has_stream_handler:
-                frmtr = logging.Formatter("[%(asctime)s] %(levelname)s\\%(name)s - %(message)s")
-                console_handler = logging.StreamHandler()
-                console_handler.setFormatter(frmtr)
-                logger.addHandler(console_handler)
+            logger = logging.getLogger("Unmock Reporter")
+            logger.setLevel(logging.DEBUG)
```

Comment:
Is there some reason to override `logging.yaml` logging level here? Might be clearer to have it one place 🤔

Reply:
Yeah, I didn't want to spam the debug log with stuff from either libraries, so I set their default level to warning and ours to debug.
Solved this in the new commit, so we don't override `root` logging configuration, only for `unmock.*` :)

Reply:
![image](https://user-images.githubusercontent.com/12184618/54415382-c8740a00-4704-11e9-92f1-6045bcb38c36.png)